src/EFTools/DesignXmlCore/VisualStudio/NativeMethods.cs

```diff
@@ -865,18 +863,18 @@ internal struct TOOLINFO
         {
             public void SetSize()
             {
-                Debug.Assert(Marshal.SizeOf(typeof(TOOLINFO)) == 40);
-                cbSize = 40;
+                cbSize = Marshal.SizeOf(typeof(TOOLINFO));
```

Comment:
I have confirmed the size computed here matches the C++ compiler when building against x64.